framework/resources/Datasources/ctas/DRILL-1248/ctas.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+source conf/drillTestConfig.properties
+
+hadoop fs -mkdir -p /drill/testdata/DRILL-1248
+
+hadoop fs -put ${DRILL_TEST_DATA_DIR}/Datasources/ctas/DRILL-1248/group_by.tsv /drill/testdata/DRILL-1248
+
+if [ -z "$PASSWORD" ]
+then
+  ${DRILL_HOME}/bin/sqlline -n ${USERNAME} -u "jdbc:drill:schema=dfs.$schema_name;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}" --run=${DRILL_TEST_DATA_DIR}/Datasources/ctas/DRILL-1248/group_by.ddl
```

Comment:
Sorry, can you explain where the $schema_name is set?

Reply:
$schema_name is in the framework/resources/Functional/group_by_alias/group_by_alias.json
"schema": "dfs.drillTestDir"

Reply:
Sorry still not clear - I mean where is it exported? I understand that the "schema_name" here has same value as "schema" in test def file.. but how does this script get that reference? Sorry could be a silly question - I guess I'm missing something